src/ScanPatron/ScanForm.js

```diff
@@ -2,66 +2,132 @@ import React from 'react';
+    scannedPatronDetails,
+    patronRRAPermission,
+    resources,
+    resetDetails,
+  } = props;
+  const isUserProfilePicConfigEnabledForTenant = get(resources, 'userProfilePicConfig.records[0].enabled');
+  const displayFooter = scannedPatronDetails && patronRRAPermission;
   const intl = useIntl();
 
   const selectUser = (user) => {
     form.change('patronBarcode', user.barcode);
   };
 
+  const containerStyle = {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    height: '100%',
+    width: '100%',
+    position: 'absolute',
+  };
+
   return (
     <form onSubmit={handleSubmit}>
-      <Row>
-        <Col xs={10}>
-          <Field
-            id="patronBarcode"
-            name="patronBarcode"
-            component={TextField}
-            placeholder={intl.formatMessage({ id : 'ui-reading-room.scanOrEnterPatronBarcode' })}
-          />
-        </Col>
-        <Col xs={2}>
-          <Button
-            type="button"
-            onClick={() => {
-              const formState = form.getState();
-              const patronBarcode = formState?.values?.patronBarcode;
-              handleScanPatron(patronBarcode);
-            }}
-          >
-            <FormattedMessage id="ui-reading-room.enter" />
-          </Button>
-        </Col>
-      </Row>
-      <Row>
-        <Col xs={12}>
-          <Pluggable
-            data-testid="clickableFindPatronPluggable"
-            aria-haspopup="true"
-            type="find-user"
-            id="clickable-find-user"
-            {...props}
-            searchLabel={<FormattedMessage id="ui-reading-room.patronLookup" />}
-            marginTop0
-            searchButtonStyle="link"
-            dataKey="patrons"
-            selectUser={selectUser}
+      <div style={containerStyle}>
+        <Paneset static>
+          <Pane
+            id="reading-room"
+            paneTitle={<FormattedMessage id="ui-reading-room.scanPatronCard" />}
+            centerContent
+            defaultWidth="fill"
           >
-            <FormattedMessage id="ui-reading-room.findUserPluginNotAvailable" />
-          </Pluggable>
-        </Col>
-      </Row>
+            <Row>
+              <Col xsOffset={2} xs={10}>
+                <Row>
+                  <Col xs={8}>
+                    <Field
+                      id="patronBarcode"
+                      name="patronBarcode"
+                      component={TextField}
+                      placeholder={intl.formatMessage({ id : 'ui-reading-room.scanOrEnterPatronBarcode' })}
+                    />
+                  </Col>
+                  <Col xs={1}>
+                    <Button
+                      type="submit"
+                      onClick={() => {
+                        const formState = form.getState();
+                        const patronBarcode = formState?.values?.patronBarcode;
+                        handleScanPatron(patronBarcode);
+                      }}
```

Comment:
Is it possible to move this function from `return` statement to get rid possible unnecessary rerenders?